.gitignore

```diff
@@ -26,6 +26,8 @@ sftp-config.json
 *.tmproject
 tmtags
 target/
+kfs-core/target
+kfs-web/target
```

Comment:
scope provided on help is to keep it from being packaged in the war since it was already overlaid to static